configure_check_conditions.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Add here, as discussed - an overview and explanation of the different scenarios this guide demonstrates, with links to their headings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Not sure why. the links suppose to be rendered in the side. Also not sure what to explain about the different scenarios that isn't the heading itself (edit conditions - what there is to elaborate about the scenario?)

Reply:
<p>No, links for inside. We talked about the user coming here - either to add/edit/etc. existing conditions in a suite</p><p>or for adding a custom conditino</p><p> </p><p>Two different scenarios... we want them to be able to understand what this notebook does and jump there quickly</p>